catalog/airflow/README.md

```diff
@@ -0,0 +1,3 @@
+### 1. 介绍
```

Comment:
We should enrich the content of README, this is way too simple.

Reply:
The doc is added now